### ISSUE TITLE
As an admin I'd like to give a reason for event rejection

### ISSUE THREAD

**Issue by svpernova09** [joindin/joindin-web2#851]
Admins should be able to:
* Click Reject button and either new page / modal appears to write a reason, optional have stock rejection reasons for selection.
* Event host receives email w/ reason and explaining why the event was rejected.

**Comment by e3betht** [comment]
Yes! This would be so helpful. Stock responses I’d like included are:

- The event needs to include a description in English.
- This is a duplicate event submission. 
- This event does not meet the requirements of being a community-focused event.
- This system is for collecting feedback for speakers. Your event does not include speakers. 

I would also like to include a way to not send a rejection email for spam events.

**Comment by svpernova09** [comment]
We could just do the rejection as a “mark as spam” option until we fleshed our that feature more (spam handling)

--
- Joe Ferguson
JoeFerguson.me
osmihelp.org

**Comment by e3betht** [comment]
It would also be good if the rejection email included the submitted event information in it so that people can easily resubmit the info with corrections.
